packages/ingest/test/test-FtpProviderClientFailure.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const test = require('ava');
+const JSFtp = require('jsftp');
+
+const FtpProviderClient = require('../FtpProviderClient');
+
+const executeRawFtpCommand = async (cmd) => {
+  const jsftp = new JSFtp({
+    host: '127.0.0.1',
+    port: 21,
+    user: 'testuser',
+    pass: 'testpass'
+  });
+  return new Promise((resolve, reject) => {
+    jsftp.raw(cmd, (err, data) => {
+      if (err) reject(err);
+      resolve(data);
+    });
+  });
+};
+
+test.before(async () => {
+  // await when fixed
+  executeRawFtpCommand('chmod 400 -R forbidden/file.txt');
+});
+
+test.after.always(async () => {
+  // await when fixed
+  executeRawFtpCommand('chmod 644 -R forbidden/file.txt');
+});
+
+test.skip('FtpProviderClient throws an error when listing a non-permitted directory', async (t) => {
```

Comment:
I assume you don't want to `skip` this test?

Reply:
I do because we can't actually set the permissions on `forbidden` to set up this test correctly.